content/applications/productivity/mail_plugins/gmail.rst

```diff
@@ -2,94 +2,160 @@
 
-Install the Gmail Plugin
-------------------------
+.. image:: gmail/gmail-side-panel.png
+   :align: center
+   :alt: Plus sign icon on the Gmail inbox side panel.
+
+Then, use the search bar to search for `Odoo` and locate the :guilabel:`Odoo Inbox Addin`.
+
+.. image:: gmail/google-workspace-marketplace.png
+   :align: center
+   :alt: Odoo Inbox Addin on Google Workspace Marketplace.
+
+Or, go directly to the :guilabel:`Odoo Inbox Addin` page on the `Google Workspace Marketplace
+<https://workspace.google.com/marketplace/app/odoo_inbox_addin/873497133275>`_.
+
+Once the plugin is located, click :guilabel:`Install`. Then click :guilabel:`Continue` to start
+the installation. 
+
+Next, select which Gmail account the user wishes to connect to Odoo. Then click :guilabel:`Allow`
+to let Odoo access the Google account. Google will then show a pop-up window confirming that the
+installation was successful.
+
+Configure the Odoo database
+---------------------------
 
-#. Open the `Gmail Plugin Apps Script project
-   <https://script.google.com/d/1n7cxtaR4fGXKcP0RwinNQmL8S4FhVqpo-ZZ_cUAhYuuDpZAP_CnHE_7q/edit>`_.
+The :guilabel:`Mail Plugin` feature must be enabled in the Odoo database in order to use the Gmail
+Plugin. To enable the feature, go to :menuselection:`Settings --> General Settings`. Under
+:guilabel:`Integrations`, activate :guilabel:`Mail Plugin`, and then click :guilabel:`Save`.
 
-#. Verify you are logged in using the Google account you want to install the plugin on.
+.. image:: gmail/mail-plugin-setting.png
+   :align: center
+   :alt: The Mail Plugin feature in the Settings.
 
-#. Click on *Publish* then *Deploy from manifest...*
+Configure the Gmail inbox
+-------------------------
 
-   .. image:: gmail/deploy-from-manifest.png
-      :align: center
-      :alt: Deploying from manifest the Gmail Plugin from the Apps Script project
+In the Gmail inbox, a purple Odoo icon is now visible on the right side panel. Click on the Odoo
+icon to open up the Odoo plugin window. Then, click on any email in the inbox. Click
+:guilabel:`Authorize Access` in the plugin window to grant Odoo access to the Gmail inbox.
 
-   .. important::
-      Make sure you are using the legacy editor; otherwise the *Deploy from manifest* functionality
-      may not be available.
+.. image:: gmail/authorize-access.png
+   :align: center
+   :alt: The Authorize Access button in the right sidebar of the Odoo plugin panel.
+
+Next, click :guilabel:`Login`. Then, enter the URL of the Odoo database that the user wishes to
+connect to the Gmail inbox, and log in to the database.
+
+.. note::
+   Use the general URL for the database, not the URL of a specific page in the database. For
+   example, use `https://mycompany.odoo.com`, not
+   `https://mycompany.odoo.com/web#cids=1&action=menu`.
```

Comment:
```suggestion
   example, use `https://mycompany.odoo.com`, not `https://mycompany.odoo.com/web#cids=1&action=menu`.
```

This line break was a little early! You can fit all of this into one line and it should be under 100 characters.

Reply:
It's more than 100 characters, the URL gets cut off.